repository/kafka/pom.xml

```diff
@@ -46,6 +46,10 @@
       <groupId>org.springframework</groupId>
       <artifactId>spring-context</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Minor: Looks like this is only used in tests. Maybe just scope to test?

Reply:
Actually removing all the guava deps and using the hashmap approach to remove downstream issues of an additional dependency.